paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/PaymentSelectionUpdater.kt

```diff
@@ -57,7 +57,7 @@ internal class DefaultPaymentSelectionUpdater @Inject constructor() : PaymentSel
                 state.linkState != null
             }
             is PaymentSelection.ExternalPaymentMethod -> {
-                true
+                state.paymentMethodMetadata.isExternalPaymentMethod(selection.type)
```

Comment:
Can you add a test for this? https://github.com/stripe/stripe-android/blob/09245bff85a9fe09b80392d4fb0a468879a76cee/paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/PaymentSelectionUpdaterTest.kt#L24